File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/examples-js/Environments_Get.js
const { DevTestLabsClient } = require("@azure/arm-devtestlabs");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Get environment.
 *
 * @summary Get environment.
 * x-ms-original-file: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/examples/Environments_Get.json
 */
async function environmentsGet() {
  const subscriptionId = "{subscriptionId}";
  const resourceGroupName = "resourceGroupName";
  const labName = "{labName}";
  const userName = "@me";
  const name = "{environmentName}";
  const credential = new DefaultAzureCredential();
  const client = new DevTestLabsClient(credential, subscriptionId);
  const result = await client.environments.get(resourceGroupName, labName, userName, name);
  console.log(result);
}

environmentsGet().catch(console.error);

File: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/examples-java/Environments_Get.java
/** Samples for Environments Get. */
public final class Main {
    /*
     * x-ms-original-file: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/examples/Environments_Get.json
     */
    /**
     * Sample code: Environments_Get.
     *
     * @param manager Entry point to DevTestLabsManager.
     */
    public static void environmentsGet(com.azure.resourcemanager.devtestlabs.DevTestLabsManager manager) {
        manager
            .environments()
            .getWithResponse(
                "resourceGroupName", "{labName}", "@me", "{environmentName}", null, com.azure.core.util.Context.NONE);
    }
}
